src/components/CohortBuilder/Categories/ActionCategory.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { noop } from 'lodash';
```

Comment:
Following our discussion on lodash vs tree-shaking : wouldn't it be more appropriate to do import noop from 'lodash/noop'; ?

Reply:
Yes, but it only pays if it's done everywhere, so it's not an issue to leave it this way for the moment.

Reply:
Bah, done it anyway :p